src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -869,7 +882,11 @@ private void printSpaces(final PrintStream stream, final int numSpaces) {
         stream.print(sb);
     }
 
-    private void handleOptionAnnotation(final Field field) {
+    /**
+     * @param field
```

Comment:
If you want to have the param annotation please fill it out. It's fine if you don't have it as this is a private method.